src/common/subsys.h

```diff
@@ -67,6 +67,7 @@ SUBSYS(throttle, 1, 1)
 SUBSYS(refs, 0, 0)
 SUBSYS(compressor, 1, 5)
 SUBSYS(bluestore, 1, 5)
+SUBSYS(seastore, 1, 5)
```

Comment:
might want to guard this line with `#ifdef WITH_SEASTAR`.

Reply:
I think this is fine without the ifdef.